arangod/Cluster/ClusterInfo.cpp

```diff
@@ -1221,14 +1222,11 @@ std::vector<std::shared_ptr<LogicalView>> const ClusterInfo::getViews(
 
   // iterate over all collections
   DatabaseViews::const_iterator it2 = (*it).second.begin();
-  while (it2 != (*it).second.end()) {
-    char c = (*it2).first[0];
-
-    if (c < '0' || c > '9') {
```

Comment:
why not change this to:
if (c >= '0' && c <= '9')

since the only keys that can start with a number are LogicalDataSource::id() and they are guaranteed to be unique